base/gmp.jl

```diff
@@ -113,57 +115,56 @@ module MPZ
 #   and `add!(x, a) = add!(x, x, a)`.
 using Base.GMP: BigInt, Limb
 
-const mpz_t = Ptr{BigInt}
-const bitcnt_t = Culong
+const Mpz_t = Ref{BigInt}
```

Comment:
No problem; I just picked the names from the GMP docs, but for other things I had been requested to use Julia naming style, like `MPZ` instead of `mpz` for the module name.